docs/advanced/event-filters.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: "Event Filters"
+---
+
+The Flare event feeds allow for searching and filtering using a `type` parameter that defines which
```

Comment:
Since we're talking about the feed parameter, should we use `types` instead of `type`?